oio/common/http.py

```diff
@@ -26,8 +26,8 @@
 
 urllib3 = patcher.import_patched('urllib3.__init__')
 
-CONNECTION_TIMEOUT = 2.0
-READ_TIMEOUT = 30.0
+CONNECTION_TIMEOUT = 10.0
```

Comment:
CONNECTION_TIMEOUT is also exposed by oio/io.py, but http.CONNECTION_TIMEOUT seems only used by oio/base.py  (but they have now have different values).